docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -174,11 +179,68 @@ For example, to exclude the :file:`sensitive` folder within the ``playbooks`` fo
      - playbooks/sensitive
      - '*.tar.gz'
 
-For more information on the :file:`galaxy.yml` file, see :ref:`collections_galaxy_meta`.
-
 .. note::
      The ``build_ignore`` feature is only supported with ``ansible-galaxy collection build`` in Ansible 2.10 or newer.
 
+Manifest Directives
+^^^^^^^^^^^^^^^^^^^
+
+With the 2.14 release, the :file:`galaxy.yml` file now supports manifest directives that are historically used in Python packaging, as described `here <https://packaging.python.org/en/latest/guides/using-manifest-in/#manifest-in-commands>`_.
+
+For example to exclude the :file:`sensitive` folder within the ``playbooks`` folder as well any ``.tar.gz`` archives, set the following in your :file:`galaxy.yml` file:
+
+.. code-block:: yaml
+
+   manifest_directives:
```

Comment:
re: the thing I mentioned about having an "escape hatch" for cases where our defaults might exclude something, what if there were two mutually-exclusive versions of the key, like:
`manifest_directives` and `manifest_directives_no_inherit` or `_no_default` or something... Naming things is hard- probably makes sense for the non-default version to be longer rather than something like `additional_manifest_directives` or `user_manifest_directives` for the case we hope people will use.

Reply:
I'll need to think about it.  Naming is hard, but I think it's an important feature.

Reply:
Done.  See `manifest_directives_full`